docs/content/posts/2023-03-09-release-1.17.0.md

```diff
@@ -0,0 +1,486 @@
+---
+authors:
+- LeonardXu:
+  name: "Leonard Xu"
+  twitter: Leonardxbj
+date: "2023-03-09T08:00:00Z" #FIXME: Change to the actual release date, also the date in the filename, and the directory name of linked images
+subtitle: ""
+title: Announcing the Release of Apache Flink 1.17
+aliases:
+- /news/2023/03/09/release-1.17.0.html #FIXME: Change to the actual release date
+---
+
+The Apache Flink PMC is pleased to announce Apache Flink release 1.17.0. Apache
+Flink is the leading stream processing standard, and the concept of unified
+stream and batch data processing is being successfully adopted in more and more
+companies. Thanks to our excellent community and contributors, Apache Flink
+continues to grow as a technology and remains one of the most active projects in
+the Apache Software Foundation. Flink 1.17 had 172 contributors enthusiastically
+participating and saw the completion of 7 FLIPs and 600+ issues, bringing many
+exciting new features and improvements to the community.
+
+
+# Towards Streaming Warehouses
```

Comment:
```suggestion
# Towards Streaming Warehouse
```

Reply:
It's better to use plural form here to express multiple warehouses

Reply:
Warehouse mean the concept itself. There should be one unique streaming warehouse. There is any description in the upcoming content pointing out multiple warehouses (multiple concepts)?

Reply:
You can refer release 1.16  as well